subsys/bluetooth/mesh/cfg_cli.c

```diff
@@ -609,13 +602,65 @@ static int mod_member_list_handle(struct bt_mesh_msg_ctx *ctx,
 	return 0;
 }
 
+static int mod_app_list_handle(struct bt_mesh_msg_ctx *ctx, struct net_buf_simple *buf, uint16_t op,
+			       bool vnd)
+{
+	struct mod_member_list_param *param;
+	uint16_t elem_addr, mod_id, cid;
+	uint8_t status;
+
+	status = net_buf_simple_pull_u8(buf);
+	elem_addr = net_buf_simple_pull_le16(buf);
+	if (vnd) {
+		cid = net_buf_simple_pull_le16(buf);
+	}
+
+	mod_id = net_buf_simple_pull_le16(buf);
+
+	if (bt_mesh_msg_ack_ctx_match(&cli->ack_ctx, op, ctx->addr, (void **)&param)) {
+
+		if (param->elem_addr != elem_addr || param->mod_id != mod_id ||
+		    (vnd && param->cid != cid)) {
+			LOG_WRN("Model Member List parameters did not match");
+			return -ENOENT;
+		}
+
+		if (param->member_cnt && param->members) {
+			int i;
+
+			for (i = 0; i < *param->member_cnt && buf->len >= 3; i += 2) {
+				key_idx_unpack(buf, &param->members[i], &param->members[i + 1]);
+			}
+
+			if (i < *param->member_cnt && buf->len == 2) {
```

Comment:
The same here. Is it worth to handle case with single index(underpacked 2 bytes) in unpack function?